.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @opengovsg/isomer-engineers
+* @karrui
```

Comment:
why not add within the isomer-engineers team?

Reply:
@karrui don't want

Reply:
hmm because of?

Reply:
usually we do add/remove within the team. not sure if there's any other reason

Reply:
oh i didn't know so troublesome, just do it then LOL

Reply:
ok i'll add you to the team and close this PR